app/qml/main.qml

```diff
@@ -402,7 +402,23 @@ ApplicationWindow {
     Connections {
         target: __loader
         onLoadingStarted: projectLoadingScreen.visible = true
-        onLoadingFinished: projectLoadingScreen.visible = false
+        onLoadingFinished:
+        {
+            projectLoadingScreen.visible = false;
+            if ( invalidLayers.length > 0 )
+            {
+                let warningMessage = qsTr( "WARNING: the following layers are invalid: " );
+                for ( let i = 0; i < invalidLayers.length - 1; ++i )
+                {
+                    warningMessage += qsTr( "'%1' " ).arg( invalidLayers[i] );
+                }
+                if ( invalidLayers.length > 1 )
+                    warningMessage += qsTr( "and '%1' " ).arg( invalidLayers[invalidLayers.length - 1] );
+                else
+                    warningMessage += qsTr( "'%1' " ).arg( invalidLayers[0] );
```

Comment:
- can you post a screenshot when there is more layers (so how it behaves when overflowing).
- Also I think we had some "warning message panel", doesn't map has other function than "notify"?

Reply:
The overflow is handled correctly:
![warning-not-overflowing resized](https://user-images.githubusercontent.com/29183781/142911232-6c943db6-a30e-47d8-9396-e543747d545f.png)